scss/_variables.scss

```diff
@@ -1056,20 +1056,23 @@ $form-check-inline-margin-end:    1rem !default;
 // scss-docs-end form-check-variables
 
 // scss-docs-start form-switch-variables
-$form-switch-width:               $spacer * 3 !default;
+// Boosted mod: no $form-switch-color
+$form-switch-width:               $spacer * 3 !default; // Boosted mod
```

Comment:
I'm not sure about the boosted mod here ?

Reply:
https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L928 2em in bootstrap